middlewares/src/main/java/com/jinyframework/middlewares/cors/Cors.java

```diff
@@ -136,20 +165,41 @@ private static void handleActual(Context ctx, Config config) {
         }
     }
 
-    private static boolean isAllowedOrigin(String origin, Config config) {
-        return config.allowOrigins.stream().anyMatch(org -> org.equalsIgnoreCase(origin));
+    private static boolean isAllowedMethod(String method, List<String> allowMethods) {
+        for (String am : allowMethods) {
+            if (am.equalsIgnoreCase(method)) {
+                return true;
+            }
+        }
+        return false;
     }
 
-    private static boolean isAllowedHeaders(List<String> reqHeaders, Config config) {
-        if (reqHeaders.get(0).isEmpty()
-                || config.allowHeaders.isEmpty()
-                || config.allowHeaders.contains("*")) {
+    private static boolean isAllowedOrigin(String origin, List<String> allowOrigins) {
+        for (String org : allowOrigins) {
+            if (org.equalsIgnoreCase(origin)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    private static boolean isAllowedHeaders(String[] reqHeaders, List<String> allowHeaders) {
+        if (reqHeaders[0].isEmpty()) {
             return true;
         }
-        return reqHeaders.stream()
-                .allMatch(header -> config.allowHeaders
-                        .stream()
-                        .anyMatch(header::equalsIgnoreCase));
+        for (String header : reqHeaders) {
+            boolean allow = false;
```

Comment:
Please use `isSomething` for `boolean` variable, it's more declarative
Expected: `allow` -> `isAllowed`

Reply:
Please consider using` lombol.var`